pkgs/terraform-config-inspect/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "terraform-config-inspect";
+  version = "20210427";
+  owner = "davido-nw";
+  rev = "v${version}";
+
+  src = fetchFromGitHub {
+    inherit owner rev;
+    repo = pname;
+    sha256 = "1x9ifpk28r9hldpvig92zsvrl5hxzc0gj75q5xr20zb3jb22q11n";
+  };
+
+  vendorSha256 = "14r0q13hi8x7h6kkb26i8v1ialwk86ykgmn53i7874w1kvmzx2hg";
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X github.com/${owner}/${pname}/cmd.Version=${rev}"
```

Comment:
I would just make sure that these flags are appropriate to this pkg but other than that it looks good 👍

Reply:
So I might as well ask cause I saw these for the `sshcb` ones - what was the purpose of needing `-s` and `-w` for that?
```
  -s    disable symbol table
...
  -w    disable DWARF generation
```

Reply:
Its been a while but I think this had to do with optimizations to the binary size by disabling both of these

Reply:
Building it without `-s` and `-w`:
```
vagrant@nix:~/nix-garage$ ls -l result/bin/
total 5704
-r-xr-xr-x 1 vagrant vagrant 5837816 Jan  1  1970 terraform-config-inspect
```
Building it with those flags:
```
vagrant@nix:~/nix-garage$ ls -l result/bin/
total 5336
-r-xr-xr-x 1 vagrant vagrant 5461912 Jan  1  1970 terraform-config-inspect
```
The end result still works fine, so keeping these flags isn't really hurting anything as far as I can tell.

Reply:
I dont think this package is leveraging `-X github.com/${owner}/${pname}/cmd.Version=${rev}` I think we can remove it from the build flags